src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -181,9 +181,10 @@ def init_db():
     Connects to DB
     :return: db_conn
     """
+    SonicDBConfig.load_sonic_global_db_config()
```

Comment:
>load_sonic_global_db_config [](start = 18, length = 27)

Is it working for single ASIC use case?

Reply:
Yes, this works for single asic as well.